plugins/pom-analyzer/src/main/java/eu/f4sten/pomanalyzer/utils/DatabaseUtils.java

```diff
@@ -82,7 +82,7 @@ public void insertIntoDB(PomAnalysisResult r, MetadataDao dao) {
             var depProduct = dep.groupId + Constants.mvnCoordinateSeparator + dep.artifactId;
             final var depId = dao.insertPackage(depProduct, Constants.mvnForge);
             var json = jsonUtils.toJson(dep);
-            dao.insertDependency(packageVersionId, depId, dep.getVersionConstraints(), null, null, null, json);
+            dao.insertDependency(packageVersionId, depId, dep.getVersionConstraintsArr(), null, null, null, json);
```

Comment:
This method is deprecated. Maybe we should remove the deprecation annotation from the method in `core`.

Reply:
Agree. This seems to be part of a larger refactoring though that we should do. Maybe we could leave it out from this PR.